faunadb-java-dsl/src/main/java/com/faunadb/client/types/Result.java

```diff
@@ -224,4 +224,13 @@ private Result() {
    */
   public abstract <U> Result<U> flatMap(Function<T, Result<U>> fn);
 
+  /**
+   * Extracts the resulting value or returns null if the operation has failed.
+   *
+   * @return the result value or null
+   */
+  public T orNull() {
+    return getOptional().orNull();
```

Comment:
unit test?

Reply:
Agreed. Although, in this case, it isn't crucial. It's a syntax sugar for the optional type, which comes from guava so, that is tested.

Reply:
ok, lets merge this sooner than later, because Brandon is removing guava optionals in another PR